docs/app/pages/getting-started/snippets/cli.txt

```diff
@@ -1,4 +1,5 @@
 "styles": [
     "./node_modules/bootstrap/dist/css/bootstrap.css",
+    "./node_modules/@ux-aspects/ux-aspects/styles/hpe-icons.css",
```

Comment:
I think we should remove this. It is a deprecated iconset, we don't want to encourage people to add it in unnecessarily.